spec/adapters/whats_app_adapter/three_sixty_dialog_inbound_spec.rb

```diff
@@ -530,6 +530,17 @@
           expect(subject).to have_received(:call)
         end
       end
+
+      describe 'sending answer request keyword' do
+        before do
+          answer_request_keyword = organization.whats_app_quick_reply_button_text['answer_request']
+          whats_app_message[:messages].first[:text][:body] = answer_request_keyword
+        end
```

Comment:
We have different test styles :slightly_smiling_face: I would do it as such:

```suggestion
      describe 'if the contributor responds with the answer request keyword' do
        before do
          organization.whats_app_quick_reply_button_text['answer_request']= 'Antworten'
          organization.save

          whats_app_message[:messages].first[:text][:body] = 'Antworten'
        end
```

I prefer to hard-code these values because they give a better idea `answer_request` could be.

This is obviously nothing for which I would press "request changes".

Reply:
I had that as well. There's no need to update the organization as its default is `Antworten`, but that does rely on the reader of the code having this knowledge.